### ISSUE TITLE
Springboot2 monitoring problems

### ISSUE THREAD

**Issue by GoogleQian** [netdata/go.d.plugin#316]
Spring boot2 (2. X) monitoring problem. I changed the original gradle packaging method, used maven, and added the same dependency，Here is docker PS
![image](https://user-images.githubusercontent.com/38888609/71872652-0f316580-3158-11ea-8808-a42b0d303962.png)，http://127.0.0.1:8080/actuator/prometheus has a return value
![image](https://user-images.githubusercontent.com/38888609/71872803-718a6600-3158-11ea-83a8-4b7c34672273.png)
Why my monitor panel doesn't have springboot2 local and springboot2 Filter，
Thank you for your help and look forward to your reply！

**Comment by ilyam8** [comment]
Hi @GoogleQian 

Could you show the output

> ./go.d.plugin -d -m springboot2

**Comment by GoogleQian** [comment]
![image](https://user-images.githubusercontent.com/38888609/71944818-ed8bb900-31ff-11ea-9662-d2429e9d8363.png)
This problem was previously encountered in docker up. I added the go global agent configuration to dockerfile.dev, and it ran successfully. However, in this case, the direct command running must not run without the agent, so it was not successful. I come from China, and the network of other countries was blocked. However, after I set the go agent configuration with docker up, the console can see the related monitoring of springboot2. I When I changed the Maven version of springboot2, I had a problem. Everything depends on the same thing, which made me wonder
